From a6a6831614d2fbc2117bd40a538af7349ae076f1 Mon Sep 17 00:00:00 2001
From: Evgeni Golov <evgeni@debian.org>
Date: Sun, 13 Dec 2020 12:24:18 +0100
Subject: [PATCH] use /etc/default, not /etc/sysconfig on Debian

---
 functions | 6 +++---
 tuned/consts.py    | 2 +-
 2 files changed, 4 insertions(+), 4 deletions(-)

--- a/functions
+++ b/functions
@@ -217,7 +217,7 @@
 CPUSPEED_SAVE_FILE="${STORAGE}/cpuspeed${STORAGE_SUFFIX}"
 CPUSPEED_ORIG_GOV="${STORAGE}/cpuspeed-governor-%s${STORAGE_SUFFIX}"
 CPUSPEED_STARTED="${STORAGE}/cpuspeed-started"
-CPUSPEED_CFG="/etc/sysconfig/cpuspeed"
+CPUSPEED_CFG="/etc/default/cpuspeed"
 CPUSPEED_INIT="/etc/rc.d/init.d/cpuspeed"
 # do not use cpuspeed
 CPUSPEED_USE="0"
--- a/tuned/consts.py
+++ b/tuned/consts.py
@@ -64,7 +64,7 @@
 SYSTEMD_CPUAFFINITY_VAR = "CPUAffinity"
 
 # irqbalance plugin configuration
-IRQBALANCE_SYSCONFIG_FILE = "/etc/sysconfig/irqbalance"
+IRQBALANCE_SYSCONFIG_FILE = "/etc/default/irqbalance"
 
 # acpi plugin configuration
 ACPI_DIR = "/sys/firmware/acpi"
