From 6768dc5befacf4c05846157a2927a54ee5e48ac5 Mon Sep 17 00:00:00 2001
From: Adriaan Schmidt <adriaan.schmidt@siemens.com>
Date: Thu, 10 Mar 2022 08:19:36 +0000
Subject: [PATCH] debian: path to grub.cfg

---
 tuned/consts.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tuned/consts.py b/tuned/consts.py
index 06e7cac..d8d35bb 100644
--- a/tuned/consts.py
+++ b/tuned/consts.py
@@ -26,7 +26,7 @@ ERROR_THRESHOLD = 3
 
 # bootloader plugin configuration
 BOOT_DIR = "/boot"
-GRUB2_CFG_FILES = ["/etc/grub2.cfg", "/etc/grub2-efi.cfg"]
+GRUB2_CFG_FILES = ["/boot/grub/grub.cfg"]
 GRUB2_CFG_DIR = "/etc/grub.d"
 GRUB2_TUNED_TEMPLATE_NAME = "00_tuned"
 GRUB2_TUNED_TEMPLATE_PATH = GRUB2_CFG_DIR + "/" + GRUB2_TUNED_TEMPLATE_NAME
-- 
2.35.1

